cmd/soroban-cli/src/config/locator.rs

```diff
@@ -253,7 +261,22 @@ impl Args {
         res
     }
 
-    pub fn remove_identity(&self, name: &str) -> Result<(), Error> {
+    pub fn remove_identity(&self, name: &str, global_args: &global::Args) -> Result<(), Error> {
+        let printer = Print::new(global_args.quiet);
+        let identity = self.read_identity(name)?;
+        if let Secret::SecureStore { entry_name } = identity {
+            let entry = StellarEntry::new(&entry_name)?;
+            match entry.delete_password() {
+                Ok(()) => {}
+                Err(e) if e.to_string() == keyring::Error::NoEntry.to_string() => {
```

Comment:
```suggestion
                Err(keyring::Error::NoEntry) => {
```
Is `e` equal to the the error? Does this work?

Reply:
Yeah, this line is a bit weird, thanks for reminding me! The thing I was running into is that `e` an error wrapped in custom errors twice, if that makes sense => locator::Error -> signer::keyring::Error -> keyring::Error. So, matching directly on `keyring::Error::NoEntry` wasn't working.

I just pushed up an update so we're not comparing the error strings. Lmk what you think!